src/libYARP_robotinterface/src/yarp/robotinterface/impl/XMLReaderFileV3.cpp

```diff
@@ -56,6 +56,7 @@ class yarp::robotinterface::impl::XMLReaderFileV3::Private
     yarp::robotinterface::Action readActionTag(TiXmlElement* actionElem, yarp::robotinterface::XMLReaderResult& result);
     yarp::robotinterface::ActionList readActionsTag(TiXmlElement* actionsElem, yarp::robotinterface::XMLReaderResult& result);
 
+    bool Check_include_section_is_enabled(const std::string& href_enable_tags, const std::string& href_disable_tags);
```

Comment:
Minor/Feel free to ignore: the rest of the class use CamelCase names for the Methods